tests/performance_tests/genesis.json

```diff
@@ -0,0 +1,23 @@
+{
+  "initial_timestamp": "2018-03-02T12:00:00.000",
+  "initial_key": "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV",
+  "initial_configuration": {
+    "max_block_net_usage": 4194304,
```

Comment:
This still limits to around ~32K I believe, I would go with 41943040 which is likely what you meant.

Reply:
Addressed: [d4c60a0](https://github.com/AntelopeIO/leap/pull/160/commits/d4c60a04349294f3df7e87e43e9fd38993b26866)